searchform.php

```diff
@@ -5,6 +5,12 @@
 ?>
 <form class="form-search" role="search" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>">
 	<div class="input-append">
-		<input type="text" placeholder="<?php _e('Search', 'largo'); ?>" class="searchbox search-query" value="<?php the_search_query(); ?>" name="s" /><button type="submit" name="search submit" class="search-submit btn"><?php _e('Go', 'largo'); ?></button>
+		<input type="text"
+		aria-label="Searchbox"
```

Comment:
This string should be translatable.

Reply:
This one could probably be more specific as well.

Reply:
All the labels in this PR are "Searchbox" so we should agree upon a single replacement.

- Sitewide Search
- Search
- ❓ are there any other suggestions?

Reply:
thanks @benlk  @joshdarby  for reviewing, i'll address your comments soon

Reply:
hi @benlk  @joshdarby , i think the `aria-label` should be the same with `placeholder`